plugin/src/main/java/com/denizenscript/denizen/objects/EntityTag.java

```diff
@@ -4135,7 +4135,7 @@ public void adjust(Mechanism mechanism) {
         // @name detonate
         // @input None
         // @description
-        // If the entity is a firework or a creeper, detonates it.
+        // If the entity is a firework, creeper, or wind charge, detonate it.

```

Comment:
`detonates` with an `s`

Reply:
Thought it was the imperative where the `s` wasn't used. Fixed in latest commit